File: VB/MySolution.Module.Mobile/Controllers/ControlViewItemControllerMobile.vb
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Layout
Imports DevExpress.ExpressApp.Mobile.MobileModel
Imports MySolution.Module.BusinessObjects

Namespace MySolution.Module.Mobile.Controllers
    Public Class ControlViewItemControllerMobile
        Inherits ObjectViewController(Of DetailView, Contact)

        Protected Overrides Sub OnActivated()
            MyBase.OnActivated()
            Dim item As ControlViewItem = TryCast(CType(View, DetailView).FindItem("MyButton"), ControlViewItem)
            If item IsNot Nothing Then
                If item.Control IsNot Nothing Then
                    item_ControlCreated(item, EventArgs.Empty)
                Else
                    AddHandler item.ControlCreated, AddressOf item_ControlCreated
                End If
            End If
        End Sub
        Private Sub item_ControlCreated(ByVal sender As Object, ByVal e As EventArgs)
            Dim button As Button = TryCast(DirectCast(sender, ControlViewItem).Control, Button)
            If button Is Nothing Then
                Return
            End If
            button.Text = "Click me!"
            button.OnClick = "DevExpress.ui.notify({ " & ControlChars.CrLf & _
                "message: 'Action from custom View Item was executed!' " & ControlChars.CrLf & _
		    "})"
        End Sub
    End Class
End Namespace

File: CS/MySolution.Module.Mobile/Controllers/ControlViewItemControllerMobile.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Layout;
using DevExpress.ExpressApp.Mobile.MobileModel;
using MySolution.Module.BusinessObjects;

namespace MySolution.Module.Mobile.Controllers
{
    public class ControlViewItemControllerMobile : ObjectViewController<DetailView, Contact>
    {
        protected override void OnActivated()
        {
            base.OnActivated();
            ControlViewItem item = ((DetailView)View).FindItem("MyButton") as ControlViewItem;
            if (item != null)
            {
                if (item.Control != null)
                {
                    item_ControlCreated(item, EventArgs.Empty);
                }
                else
                {
                    item.ControlCreated += item_ControlCreated;
                }
            }
        }
        private void item_ControlCreated(object sender, EventArgs e)
        {
            Button button = ((ControlViewItem)sender).Control as Button;
            if (button == null) return;
            button.Text = "Click me!";
            button.OnClick = @"DevExpress.ui.notify({ 
                message: 'Action from custom View Item was executed!' })";
        }
    }
}
